OpenRobertaServer/src/main/java/de/fhg/iais/roberta/javaServer/restServices/all/controller/ClientAdmin.java

```diff
@@ -25,8 +25,8 @@
 import de.fhg.iais.roberta.util.RandomUrlPostfix;
 import de.fhg.iais.roberta.util.ServerProperties;
 import de.fhg.iais.roberta.util.Statistics;
-import de.fhg.iais.roberta.util.UtilForREST;
```

Comment:
Although these formatting changes are nice, this file was not touched otherwise and you can remove these changes to keep clutter low.